File: ScannerVB6/ScannerVB6/Model/Formulaire.cs
namespace ScannerVB6.Model
{
    public class Formulaire
    {
        public required string NomDuFormulaire { get; set; }
        public required string NomDuFichierFRM { get; set; }
        public int NombreDeUserControles { get; set; } = 0;
        public int NombreDeCreateObject { get; set; } = 0;
        public int NombreDApi32 { get; set; } = 0;
        public int NombreDeExcelApp { get; set; } = 0;
        public int NombreDeWordApp { get; set; } = 0;
        public int NombreDeOfficeouAutre { get; set; } = 0;
        public int NombreDeLignesActives { get; set; } = 0;
        public List<RegexFound>? ListofRegeX { get; set; } = new List<RegexFound>();

    }
}

File: ScannerVB/Model/Formulaire.vb
Public Class Formulaire
    Property NomDuFormulaire As String
    Property NomDuFichierFRM As String
    Property NombreDeUserControles As Integer = 0
    Property NombreDeCreateObject As Integer = 0
    Property NombreDApi32 As Integer = 0
    Property NombreDeExcelApp As Integer = 0
    Property NombreDeWordApp As Integer = 0
    Property NombreDeOfficeouAutre As Integer = 0
    Property NombreDeLignesActives As Integer = 0
    Property ListofRegeX As New List(Of RegexFound)

End Class
